### PR TITLE
Documentation review

### PR DESCRIPTION
**Pull request by jonathom**
I noticed a few things while going through the "getting started" guides, some of which I already fixed here. I exchanged the `max` process for a `min` process in R & Python because a `max` doesn't result in a good visualization when dealing with SAR data. I haven't had a look at the AUTH sections other than reading it because I'm missing the tokens/accounts.

### CODE REVIEW

**Comment by jonathom** [comment]
I can't comment on lines I didn't edit here on github, so here are the remaining issues:

- [ ] JS: The code to check on the status of the batch job doesn't print anything for me (`let stopFn...`)
- [ ] Python: The example to add a process that is "not supported by the python client" seems a bit out of context. It is a) something that appears overkill to me in a "getting started" and is b) dealing with Sentinel2 bands, when the example is S1. Should be taken out IMO.

File: getting-started/r/index.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ jobs = list_jobs()
 jobs # printed as a tibble or data.frame, but the object is a list
 
 # or use the job id (in this example 'cZ2ND0Z5nhBFNQFq') as index to get a particular job overview
-jobs$cZ2ND0Z5nhBFNQFq
+list_jobs()$cZ2ND0Z5nhBFNQFq
```
Comment (m-mohr) [inline getting-started/r/index.md:219]: Why this? jobs is already defined, right?

Comment (m-mohr) [reply getting-started/r/index.md:219]:
```suggestion
jobs$cZ2ND0Z5nhBFNQFq
```

**Comment by m-mohr** [comment]
Thanks, I also ported these changes over to openeo.org.

> * [ ]  JS: The code to check on the status of the batch job doesn't print anything for me (`let stopFn...`)

I'll need to check, but maybe the back-end is not sending anything? Have you tried with GEE? I think my tests still succeed...

> * [ ]  Python: The example to add a process that is "not supported by the python client" seems a bit out of context. It is a) something that appears overkill to me in a "getting started" and is b) dealing with Sentinel2 bands, when the example is S1. Should be taken out IMO.

It's important to mention how to use other processes as, in contract to JS and R, the Python client doesn't auto-generate code for all processes based on the /processes endpoint. So it should be kept in here, but instead we could improve the example, I think. Or we separate it a bit more from the other S1 code by using these "Tip" blocks or so.

File: getting-started/r/index.md (hunk 1 above)
Comment (jonathom) [reply getting-started/r/index.md:219]: jobs is defined but a static object, so `jobs$ID` will return the same status without any updates.

Comment (m-mohr) [reply getting-started/r/index.md:219]: Yeah, but here it is only shown that this is an alternative way to get the details from the list_jobs() request, actually, earlier it says that you need to re-run list_jobs() to refresh the data. So this is good as it is IMHO.

**Comment by jonathom** [comment]
> I guess the back-end is not sending anything. Have you tried with GEE? I think my tests still succeed, so think the code is working, it just doesn't receive anything...

Yes, the logging works with GEE. It just won't print any urls to the console, the first way or the second (TIP box). If this will work with ideally implemented backends its not a problem, I was just wondering.

**Comment by jonathom** [comment]
I'm having problems running the concerning python example:
```python
datacube = datacube.process(process_id="ndvi", 
                            args={ "data": {"from_node": datacube._pg}, 
                                   "nir": "B08", 
                                   "red": "B04"})
```
putting this in front:
```python
datacube = connection.load_collection(
  "SENTINEL2_L2A_SENTINELHUB",
  spatial_extent={"west": 16.06, "south": 48.06, "east": 16.65, "north": 48.35},
  temporal_extent=["2017-03-01", "2017-04-01"],
  bands=["B04", "B08"]
)

datacube = datacube.min_time()
```
and of course a `result = datacube.save_result("GTiff")` behind, it creates this process graph:
```
{
  "process_graph": {
    "ndvi1": {
      "process_id": "ndvi",
      "arguments": {
        "args": {
          "data": {
            "from_node": null
          },
          "nir": "B08",
          "red": "B04"
        }
      }
    },
    "saveresult1": {
      "process_id": "save_result",
      "arguments": {
        "data": {
          "from_node": "ndvi1"
        },
        "format": "GTiff",
        "options": {}
      },
      "result": true
    }
  }
}
```
and fails with the error: `openeo.rest.OpenEoApiError: [500] unknown: from_node None (referenced by 'ndvi1') not in process graph.`

So how would the correct process node be referenced? Maybe @soxofaan can help as well?

**Comment by m-mohr** [comment]
> Yes, the logging works with GEE. It just won't print any urls to the console, the first way or the second (TIP box). If this will work with ideally implemented backends its not a problem, I was just wondering.

GEE logging should be "ideally implemented". What does it print for GEE? @jonathom

**Comment by jonathom** [comment]
> GEE logging should be "ideally implemented". What does it print for GEE? @jonathom

Only logs for the job, but no download URLs. With this code
```js
[...]
await job.startJob();

let stopFn = job.monitorJob(async (job, logs) => {
    console.log(job.status);
    logs.forEach(log => console.log(`${log.level}: ${log.message}`));
    if (job.status === "finished") {
        var urls = await job.listResults();
        urls.forEach(url => console.log(`Download result from: ${url}`));
    }
});

stopFn();
```
I get this output:
```
running
info: Queueing batch job
info: Starting batch job
debug: Executing node loadco1
debug: Executing node reduce1
debug: Bypassing node min1; Executing as native GEE reducer instead.
debug: Executing node savere1
warning: Compositing the image collection to a single image.
```

**Comment by m-mohr** [comment]
That's the expected behavior. So that works, great :-)

**Comment by jonathom** [comment]
> Have you waited long enough?

I'm not stopping the script manually, it just ends after the lines above are printed. 

Does it behave the same way when you run it ([gist with js code](https://gist.github.com/jonathom/93ea6ad5e247688ce8fe9f55d84f8b58))?

**Comment by m-mohr** [comment]
Sure, you called stopFn. Try without it.

**Comment by jonathom** [comment]
Alrighty. Sorry for the hubub. However it did print `Download result from: [object Object]`, so I think `urls.forEach(url => console.log('Download result from: ${url}'));` should be `urls.forEach(url => console.log('Download result from: ' + JSON.stringify(url.href)));`, at least that works for me.

**Comment by soxofaan** [comment]
> I'm having problems running the concerning python example:
```
    datacube = datacube.process(process_id="ndvi", 
                            args={ "data": {"from_node": datacube._pg}, 
                                   "nir": "B08", 
                                   "red": "B04"})
```


There are a couple of things wrong with this snippet: to set the process arguments you should use the `arguments` argument, not `args`, also it is not necessary to explicitly do `from_node` stuff or use the private `_pg` property.

This should work:
```python
datacube = datacube.process(
    process_id="ndvi", 
    arguments={
        "data": datacube, 
        "nir": "B08", 
        "red": "B04"}
)
```

It can even be done with direct arguments to the `process` method like this:
```python
datacube = datacube.process(
    process_id="ndvi", 
    data=datacube, 
    nir="B08", 
    red="B04"
)

**Comment by jonathom** [comment]
@soxofaan Thanks! Solved it right away.

@m-mohr It actually just needed `url.href`, so none of that JSON.stringify required. Now it prints correctly.
I also rearranged the python example accordingly.

**Comment by m-mohr** [comment]
Great, thanks. @jonathom Could you please also update the documents for openeo.org at the same time? They are mostly copy/pasted and should contain the same issues.

**Pull request merged**